development/build/README.md

```diff
@@ -1,6 +1,6 @@
 # The MetaMask Build System
 
-> _tl;dr_ `yarn dist` for prod, `yarn start` for local development
+> _tl;dr_ `yarn dist` for prod, `yarn start` for local development. Add `--build-type flask` to build flask (with more experimental features).
```

Comment:
We document the `--build-type` flag near the bottom of this document, but the `flask` build type is missing. Would be good to add it there as well!